README.md

```diff
@@ -13,7 +13,11 @@ Files for running SciCat with docker-compose.
    ```sh
    docker-compose up -d
    ```
-3. SciCat will now be available on http://localhost. The Loopback API explorer of catamel is available at http://localhost/explorer/, the one for the search-api at http://localhost/panosc-explorer/.
+3. SciCat will now be available on http://localhost. 
+
+The Loopback API explorer of catamel is available at http://catamel.localhost/explorer/ and the APIs at http://catamel.localhost/api/. 
```

Comment:
I think this should be "The Swagger UI for the backend is available at http://backend.localhost/explorer and the APIs at https://backend.localhost.api"